wiki/Tournaments/OWC/2/en.md

```diff
@@ -238,79 +269,334 @@ This competition has come to an end and resulted in the following podium:
+   - **Prior to the beginning of the knock-out stages**, teams that are still active in the tournament are allowed to introduce a maximum of two new players to substitute in for two other prior members of the team with the Tournament Management's approval.
+6. Mappool selectors are not eligible to participate as players in this tournament.
+
+### Match instructions
+
+1. A member of the Tournament Management will create a multiplayer room 15 minutes in advance. Players must gather during this period.
+   - Room settings are osu!, Team-Vs., Win Condition: 'Score'. Room name must be "OWC2: (TeamRed) vs (TeamBlue)".
+   - The team mentioned first in the room name must be the red team, the team mentioned second in the room name must be the blue team.
+2. As osu! World Cup #2 features no formal referee, **both team captains are expected to referee the match by themselves from inside the multiplayer lobby**. If there are any suspected irregularities during the run of the match, the concerned team captain may file a report in the discussion thread which will be investigated by the Tournament Management in due time.
+3. Map banning **does not apply** in the entirety of the tournament.
+4. Each team must have 4 players for each map. They can be exchanged freely after a map is concluded.
+5. Beatmap selection will alternate between each captain selecting a beatmap out of the mappool.
+6. Each captain must use the "!roll" command once in #multiplayer.
+   - The captain with the higher roll decides which team **picks** first.
+7. The results of eatch match will be posted on the discussion thread after the match has been concluded by either of the team captains.
 
 ### Stage instructions
 
-1. We are looking for 32 teams from 32 countries. If there are less participants than we want, the tournament will be modeled according to the reached number.
-2. In the first stage, the teams will be divided into 8 groups of 4 team in each one.
-3. All the teams from each group will face each other.
-4. 2 teams with the largest amount of points are being promoted to the 1/16 Cup Stage.
+1. In the Group Stage, the teams will be divided into 8 groups of 4 teams at random.
+   - To ensure fairness, the drawing process will be [publicly livestreamed](http://www.livestream.com/natteke).
+2. All the teams from each group will face each other over the course of three weeks.
+3. Rankings of each group are determined by sorting the results of each team's performance in the following priority:
+   - Most matches won.
+   - Have higher `{(the number of beatmaps won) - (the number of beatmaps defeated)}`.
```

Comment:
```suggestion
   - Have higher `(the number of beatmaps won) - (the number of beatmaps defeated)` value.
```

Reply:
this one is taken from OWC 2017 and OWC 2018's wiki page (the last two OWCs to feature Group Stage) so yeah i'll be keeping this for all around consistency i think

![image](https://user-images.githubusercontent.com/36564236/105469013-3955b700-5cca-11eb-8a92-24b364f395f7.png)